scripts/windows/msys2_prepare.bat

```diff
@@ -13,13 +16,16 @@ rem Set required vars defaults
 IF "%ERTS_VERSION%"=="" SET "ERTS_VERSION=9.3"
 IF "%OTP_VERSION%"=="" SET "OTP_VERSION=20.3"
 IF "%WIN_OTP_PATH%"=="" SET "WIN_OTP_PATH=C:\Program Files\erl"
+IF "%WIN_JDK_BASEPATH%"=="" SET "WIN_JDK_BASEPATH=C:\Program Files\Java"
```

Comment:
Shall these new params documented for user? See docs/build-windows.md

Reply:
Yes should be.